packages/widget-reorder/src/action.ts

```diff
@@ -0,0 +1,64 @@
+import { List } from 'immutable'
+import { extract, diff, reorder } from './utils'
+import { ReorderAction as _ } from './action.types'
```

Comment:
Not sure! I like the idea of keeping the constants grouped as enums but in this case it didn't really make sense since the reorder actions didn't group very well